opus/application/test_api/responses/api_metadata_vg_iss_2_s_c4360845_default_html.html

```diff
@@ -61,6 +61,12 @@
 <dt>Wavelength Band</dt><dd>N/A</dd>
 <dt>Signal Source Name</dt><dd>N/A</dd>
 <dt>Receiver Host Name</dt><dd>N/A</dd>
+<dt>Observed Ring Radius (Min) (km)</dt><dd>N/A</dd>
```

Comment:
These should have a [Ring] after them in the resulting metadata, not just on the search page.